scripts/util/doc-builder.js

```diff
@@ -89,11 +90,18 @@ function checkoutSrcDirectories() {
   });
 }
 
+function copyPackageJSON(apiDocsMeta) {
+  console.info('info:', 'Copying over package.json to docs/_data/package.json');
+  fsExtra.copySync('package.json', options.docsPackageJSONRelPath);
```

Comment:
if you decide you only want `version`, this can turn into something like
```
fsExtra.writeJSONSync(options.docsPackageJSONRelPath, {version: latestVersion});
```